src/Shared/Microsoft.DotNet.Scaffolding.Shared/MessageStrings.resx

```diff
@@ -172,4 +180,10 @@
   <data name="FailedAddPackage" xml:space="preserve">
     <value>Failed to add package {0}</value>
   </data>
-</root>
+  <data name="DirectoryNotEmpty" xml:space="preserve">
+    <value>The directory is not empty : '{0}'</value>
+  </data>
+  <data name="PathNotFound" xml:space="preserve">
+    <value>	Could not find a part of the path '{0}'</value>
```

Comment:
Is the spaces before `Could not...` intentional? I'd expect the indentation be managed on the user of the string, not the string itself.

Reply:
will remove that ty